### DISCUSSION TITLE
Always apply a specific plugin or set a target for a single plugin?

### DISCUSSION THREAD

**Discussion by viveleroi** [babel/babel#15003] [Q&A] [answered]
I'd like to configure babel to use preset-env for latest browsers by default, but always convert arrow-functions. That plugin is part of preset-env but doesn't apply unless I set a target that needs it, for example `ie: 11`. 

We're on a decade-old AngularJS project and a lot of our functions are written as arrow functions. The sad thing is, browsers support most of the javascript we use now but angular still needs those converted to normal functions. I don't want to use `ie: 11` for all other features in almost every case, the browsers we support work fine and we don't need to bloat our output.

**Comment by JLHwung** [comment, marked as answer]
You can specify the [`include` option](https://babeljs.io/docs/en/babel-preset-env#include) of `preset-env`:

```json
{
  "presets": [
    ["@babel/preset-env", { "include": ["@babel/plugin-transform-arrow-functions"] }]
  ]
}
```
